### PR TITLE
documentation updates 2019 06 12

### PR DESCRIPTION
**Pull request by thejuliekramer**
updates introduction.md for readability and deploy_macos with dependencies

### CODE REVIEW

File: docs/source/introduction.md [hunk 1]
```diff
@@ -58,9 +58,9 @@ Compliance is a technique humans have developed for enabling trust in systems th
 * Easy-to-use, beautiful questionnaires
 * Jargon-free approach to security controls and compliance
 * Step-by-step guidance through assessments
-* Compliance-as-Code approach to documentation
+* Compliance-as-Code approach to creating documentation
 * Discuss questions and answers in the tool instead of in email
-* Support for rich, clear multi-media communication
+* Support for rich, clear multi-channel communication
```
Comment (gregelin) [inline docs/source/introduction.md]: Even more accurate would be:

```
* Support for rich, clear multi-media and multi-channel communication
```

File: docs/source/introduction.md [hunk 2]
```diff
@@ -58,9 +58,9 @@ Compliance is a technique humans have developed for enabling trust in systems th
 * Easy-to-use, beautiful questionnaires
 * Jargon-free approach to security controls and compliance
 * Step-by-step guidance through assessments
-* Compliance-as-Code approach to documentation
+* Compliance-as-Code approach to creating documentation
```
Comment (gregelin) [inline docs/source/introduction.md]: Better would be:

```
* Compliance-as-Code approach to creating compliance documentation
```

**Comment by gregelin** [comment]
@thejuliekramer I made a couple suggestions. If you could please update, I will approve.

**Comment by thejuliekramer** [comment]
@gregelin 👍 updated based on your comments

**Pull request merged**